dts/bindings/i3c/i3c-controller.yaml

```diff
@@ -30,3 +30,9 @@ properties:
       and there are I2C devices attached to the bus, look at the Legacy
       Virtual Register (LVR) of all connected I2C devices to determine
       the maximum allowed frequency.
+
+  supports-setaasa:
+    type: boolean
+    description: |
+      Indicates if the device supports the CCC SETAASA. If true, it will
+      be used as an optimization for bus initialization.
```

Comment:
This shouldn't be added here, remove

Reply:
The reason for adding this change was an issue I encountered during compilation when adding I3C targets in the overlay file for the samples. The error in the DTS reported something like :
' error: 'DT_N_S_soc_S_i3c_40005c00_S_i3cdev0_000000020813811000_P_supports_setaasa' undeclared here...'

Upon investigating the code, I found that the issue comming from the following macro in Zephyr:

#define I3C_DEVICE_DESC_DT(node_id)
{
.bus = DEVICE_DT_GET(DT_BUS(node_id)),
.dev = DEVICE_DT_GET(node_id),
.static_addr = DT_PROP_BY_IDX(node_id, reg, 0),
.pid = ((uint64_t)DT_PROP_BY_IDX(node_id, reg, 1) << 32)
| DT_PROP_BY_IDX(node_id, reg, 2),
.init_dynamic_addr =
DT_PROP_OR(node_id, assigned_address, 0),
.supports_setaasa = DT_PROP(node_id, supports_setaasa),
}

The problem happens because DT_PROP is used to directly fetch the supports_setaasa property, but if the property is missing, the macro does not handle it.

Reply:
What is the overlay here? This really looks like you are using the wrong the include with the dts bindings for what you have in your overlay?

Reply:
@ExaltZephyr I still need a response to this

Reply:
Overlays in Zephyr are files used within samples to extend or modify the DTS configurations for a board. The compilation error occurred because a required property was missing in the I3C YAML files. I added the missing property to ensure that future sample additions won't face the same issue that I faced when I added my sample to test
ok ?

Reply:
no, not okay, I know what overlays are, and I've seen people internally see the same error that you are seeing, and it's always because of having a wrong binding in their yaml file. What sample are you referring too? I don't see anything in this PR.

There should be `include: ["i3c-device.yaml"]` within there not `include: ["i3c-controller.yaml"]`. This commit must be dropped

Reply:
The sample I'm referring to is a new one that I added locally just for testing.

Anyway, I will drop the commit now